source/computer/fedora-setup.rst

```diff
@@ -1,17 +1,220 @@
 Fedora 配置指南
 ===============
 
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-01-19
+:预计花费时间: 120 分钟
+
 .. note::
 
-   本文尚未完成。
+   本节内容适用于 **Fedora 33 Workstation**\，对其他 Fedora 版本不一定适用。
+   建议用户安装时总是选择 Fedora 最新版本。如遇到问题，欢迎反馈。
+
+----
+
+安装系统
+--------
```

Comment:
Shall we add a subsection about how to update the system?
I don't know how to do it now.

Reply:
Fedora will pop up a notification window about available updates, just like macOS. I feel that it's easier to just click the "update" button to update the system.

Reply:
Command line: https://docs.fedoraproject.org/en-US/quick-docs/dnf-system-upgrade/

Reply:
Done in ad80af17136899fcc66818842283b0d2193592a1